Dockerfile

```diff
@@ -71,57 +71,30 @@ RUN apt-get update \
     gnupg \
     lsb-release \
   && curl --silent --show-error --location https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | tee /etc/apt/trusted.gpg.d/microsoft.gpg > /dev/null \
+  # hardcoded to amd64 due to https://github.com/Azure/azure-cli/issues/7368
   && echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $(lsb_release -cs) main" | tee /etc/apt/sources.list.d/azure-cli.list \
   && apt-get update \
   && apt-get install --yes --no-install-recommends azure-cli="${AZURE_CLI_VERSION}-1~$(lsb_release -cs)" \
-  && az --version \
+  && az --version || echo 'this will currently fail on non amd64 architectures' \
```

Comment:
You might be interested in https://github.com/jenkins-infra/packer-images/blob/main/provisioning/ubuntu-provision.sh#L223-L235 : this is how we install `az` for the VM templates, which has arm64 ubuntu support (but 20.04)

Reply:
Thanks, I've tested and it works on 22.04 as well